src/libnnstreamer-edge/nnstreamer-edge-mqtt-mosquitto.c

```diff
@@ -447,7 +451,8 @@ nns_edge_mqtt_get_message (nns_edge_broker_h broker_h, void **msg,
    * The time to wait for new data, in milliseconds.
    * (Default: 0 for infinite timeout)
    */
-  if (!nns_edge_queue_wait_pop (bh->message_queue, timeout, msg, msg_len)) {
+  if (NNS_EDGE_ERROR_NONE !=
+      nns_edge_queue_wait_pop (bh->message_queue, timeout, msg, msg_len)) {
     nns_edge_loge ("Failed to get message from mqtt broker within timeout.");
     return NNS_EDGE_ERROR_UNKNOWN;
```

Comment:
You now know why. It's no longer "UNKNOWN". you can return the error code of ```nns_edge_queue_wait_pop```

Reply:
I changed return type of nns_edge_queue_wait_pop.
This is a point i missed, Thanks for comment.